src/panels/developer-tools/statistics/developer-tools-statistics.ts

```diff
@@ -0,0 +1,156 @@
+import { css, CSSResultGroup, html, LitElement } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import {
+  StatisticsValidationResult,
+  validateStatistics,
+} from "../../../data/history";
+import { haStyle } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
+import "@material/mwc-button/mwc-button";
+import "../../../components/data-table/ha-data-table";
+import type { DataTableColumnContainer } from "../../../components/data-table/ha-data-table";
+import { showAlertDialog } from "../../../dialogs/generic/show-dialog-box";
+import { showFixStatisticsUnitsChangedDialog } from "./show-dialog-statistics-fix-units-changed";
+
+@customElement("developer-tools-statistics")
+class HaPanelDevStatistics extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ type: Boolean }) public narrow!: boolean;
+
+  @state() private _data: (StatisticsValidationResult["data"] & {
+    type: string;
+  })[] = [];
+
+  protected firstUpdated() {
+    this._validateStatistics();
+  }
+
+  private _columns: DataTableColumnContainer = {
+    statistic_id: {
+      title: "Entity",
+      sortable: true,
+      filterable: true,
+      direction: "asc",
+      grows: true,
+    },
+    type: {
+      title: "Issue",
+      sortable: true,
+      filterable: true,
+      direction: "asc",
+      width: "60%",
+      template: (type, data) =>
+        html`${this.hass.localize(
+          `ui.panel.developer-tools.tabs.statistics.issues.${type}`,
+          data
+        ) || type}`,
+    },
+    fix: {
+      title: "",
+      template: (_, data) =>
+        html`<mwc-button @click=${this._fixIssue} .data=${data}
+          >Fix issue</mwc-button
+        >`,
+      width: "113px",
+    },
+  };
+
+  protected render() {
+    return html`
+      <ha-data-table
+        .columns=${this._columns}
+        .data=${this._data}
+        noDataText="No issues found!"
+      ></ha-data-table>
+    `;
+  }
+
+  private async _validateStatistics() {
+    const issues = Object.values(await validateStatistics(this.hass));
+    this._data = [];
```

Comment:
Would it be an idea to show all statistic IDs that we have ?

Reply:
We could do that... And then sort by issue?